code/game/objects/items.dm

```diff
@@ -127,6 +127,9 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 	///All items with sharpness of IS_SHARP or higher will automatically get the butchering component.
 	var/sharpness = IS_BLUNT
 
+	///All items with eyestab of TRUE can be used to poke someone's eyes out
+	var/can_eyestab = FALSE

```

Comment:
Making vars for specific behaviors like this is usually not something we'd want to do, making this into an element would be preferable.